libs/ledger-live-common/src/currencies/__snapshots__/sortByMarketcap.test.ts.snap

```diff
@@ -184,7 +184,7 @@ Array [
   "ethereum/erc20/dtr",
   "ethereum/erc20/crypto20",
   "ethereum/erc20/karatbank_coin",
-  "ethereum/erc20/ecoreal_estate",
```

Comment:
Might be problematic

Reply:
True, but it's unfortunately happening all the time in previous CAL updates... seems safe to keep doing it for now, then we should consider maybe a way to migrate tokens instead of patching the list.